### PR TITLE
BaSP-TG-25: List and delete Admins Page

### PR DESCRIPTION
**Pull request by LucasPrat1**
- [x] Table list
- [x] Delete and edit Icon
- [x] Delete feature
- [x] Redirect to add/edit form
- [x] Warning message for delete
- [x] Success message for confirm delete

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins`);
+      const data = await response.json();
+      setList(data.data);
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  // unnecessary
```
Comment (martumoser) [inline src/Components/Admins/index.js:18]: Eliminar este comment

File: src/Components/Admins/ListItem/ListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const ListItem = ({ listItem, setShowModal, setTitleModal }) => {
+  const handleDelete = async (_id) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins/${_id}`, {
+        method: 'DELETE'
+      });
+      setShowModal(true);
+      setTitleModal('Admin deleted successfully');
+      console.log('response', response);
```
Comment (martumoser) [inline src/Components/Admins/ListItem/ListItem.jsx:12]: Eliminar este console.log

File: src/Components/Admins/Modal/Modal.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './modal.module.css';
+
+const Modal = (props) => {
+  if (!props.show) {
+    return null;
+  }
```
Comment (martumoser) [inline src/Components/Admins/Modal/Modal.jsx:7]: Dejar un espacio entre estas lineas

File: src/Components/Admins/Modal/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './modal.module.css';
+
+const Modal = (props) => {
```
Comment (martumoser) [inline src/Components/Admins/Modal/Modal.jsx:4]: Destructurar las props, es decir, especificar cuales son las props que se reciben. En este caso: 
`const Modal = ({ title, closeModal, show })`

**Review by martumoser — CHANGES_REQUESTED** [review]

**Comment by NancyCima** [comment]
Good Job!

**Review by LucianoAlarcon — APPROVED** [review]

**Pull request merged**